src/Og.php

```diff
@@ -183,6 +182,166 @@ public static function getEntityGroups(EntityInterface $entity, array $states =
   }
 
   /**
+   * Returns all group IDs associated with the given group content entity.
+   *
+   * Do not use this to retrieve group IDs associated with a user entity. Use
+   * Og::GetEntityGroups() instead.
```

Comment:
should be `Og::getEntityGroups` - but with the recent changes in OgMembership we can probably call it `Og::getUserEntityGroups` - can be in follow up.